src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2200,18 +2200,10 @@ private BoundExpression BindArgumentValue(DiagnosticBag diagnostics, ArgumentSyn
 
             SourceLocalSymbol localSymbol = this.LookupLocal(argumentSyntax.Identifier);
 
-            // In error scenarios with misplaced code, it is possible we can't bind the local declaration.
-            // This occurs through the semantic model.  In that case concoct a plausible result.
             if ((object)localSymbol == null)
             {
-                localSymbol = SourceLocalSymbol.MakeLocal(
-                    ContainingMemberOrLambda,
-                    this,
-                    RefKind.None,
-                    typeSyntax,
-                    argumentSyntax.Identifier,
-                    LocalDeclarationKind.RegularVariable,
-                    initializer: null);
```

Comment:
There is identical code and comment in `LocateDeclaredVariableSymbol`. I wonder if it should also be updated.